assessments/guide/outline.md

```diff
@@ -94,15 +105,15 @@ encryption by default.)
 
```

Comment:
## Security Issue Resolution
* Responsible Disclosures Process.  A outline of the project's responsible disclosures process should suspected security issues, incidents, or vulnerabilities be discovered both external and internal to the project.  The outline should discuss communication methods/strategies.
* Incident Response.  A description of the defined procedures for triage, confirmation, notification of vulnerability or security incident, and patching/update availability.